compiler/rustc_codegen_ssa/src/back/archive.rs

```diff
@@ -56,6 +59,9 @@ pub trait ArchiveBuilderBuilder {
             if !bundled_lib_file_names.contains(&Symbol::intern(name)) {
                 continue; // We need to extract only native libraries.
             }
+            let data = search_for_section(rlib, data, ".rlib_entry").map_err(|e| {
```

Comment:
For some targets like wasm creating a wrapper file is not yet possible and `create_wrapper_file` will instead return the original file unchanged.

Reply:
Looks like `search_for_section` already handles that. Never mind.